### DISCUSSION TITLE
Domain verification issue

### DISCUSSION THREAD

**Discussion by hmjk99** [vercel/community#6191] [Help]
### Summary

I added a domain to my project and added the txt record in my dns provider. It has been over 7 hours but it is still not verified.

### Steps to Reproduce

![IMG_20240320_161340](https://github.com/vercel/community/assets/124408498/f447d9fe-7fc9-49a3-8ab1-c0d9e39284ba)
<img width="942" alt="Screenshot 2024-03-20 at 5 22 02 PM" src="https://github.com/vercel/community/assets/124408498/ff9d5430-d46c-4fdf-828d-cf9e503ba650">

**Comment by amyegan** [comment]
It looks like [your TXT record](https://dnschecker.org/#TXT/_vercel.mastergpt.fr) has not propagated globally yet. I don't see [your CNAME record](https://dnschecker.org/#CNAME/client.mastergpt.fr) either. 🤔 You may need to reach out to your domain registrar for help with this.
